management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -0,0 +1,197 @@
+between component types over the period selected. *Component's Usage* is a treemap and provides an overview of how many jobs were executed for the components. You can drill down for a particular 
+configuration or project to see how many credits it consumed. The last visual also shows the credits consumed per component split between SQL, KBC, and CDC credits. 
+
+#### Top 10 
+{: .image-popup}
+![Screenshot - Top 10](/management/telemetry/telemetry-dashboards/ac-project-overview-top10.png)
+
+Top 10 has two tables. The first is the *Top 10 Transformations* based on their consumption. It provides details about the runtime and input and output mapping (e.g., whether any filters were applied or 
+whether the output was incremental or full load). *Top 10 Component Configurations* shows the most expensive configurations and provides information on how long they ran for. 
+
+#### Significant changes (outliers)
+{: .image-popup}
+![Screenshot - Top 10](/management/telemetry/telemetry-dashboards/ac-project-overview-outliers.png)
+
+*Significant changes* shows outliers for projects, configurations, and users. The tables show the average PPU consumption over the previous 30 days for projects, configurations, or users that consumed
+50% more or 50% less on given days. It lists days on which a project billed more than 1 credit. The last column is the percentage of billed credits versus the average. 
+
+### Project Users
+The first section provides an *Overview* of users within a project. They can be filtered by branch type and specific user. It shows the user roles, activities, and how many joined the project 
+in the last seven days. 
+
+#### Overview
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-users-overview.png)
+
+The second section, *Activities*, summarizes the users in a project. 
+
+#### Activities
+{: .image-popup}
+![Screenshot - Activities](/management/telemetry/telemetry-dashboards/ac-project-users-activities.png)
+
+### Project Health
+This tab shows the overall health of a project. It can be filtered by branch (development vs. main). The *Overview* section focuses on flows and jobs. The next section focuses on trends for *Components*, 
+and the last section shows four individual metrics: Configurations without a Description*, *Flows without Error Notifications*, *Unused Configurations*, and *Unused Tables*.
+
+#### Overview
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-overview.png)
+
+#### Components
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-components.png)
+
+#### Configurations, flow notifications, and tables
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-config.png)
+
```

Comment:
This section shows important numbers with drill available for all. These are not causing immediate issues but shows information that might be useful for a clean up exercise. *Configurations without Description* shows how many configuration doesn't have description filled and therefore it is not clearly visible what the configuration does. *Flows without Error Notifications* show how many flows doesn't send any notification while end up in error -> if nobody manually checks this won't be found. *Unused Configurations* and *Unused Tables* show the configurations which haven't had any job started in the last 30 days or tables that haven't had any import or export in the last 30 days.